contribute/pull-request.md

```diff
@@ -0,0 +1,99 @@
+# Create a pull request
+
+We're excited that you're considering making a contribution to the Grafana project! This document guides you through the process of creating a [pull request](https://help.github.com/en/articles/about-pull-requests/).
+
+## Prerequisites
```

Comment:
This really isn't a prerequisite, and this seems to aimed at beginning users. I suggest something like "Before you begin."